_posts/2023-03-13-changelog-190.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #190"
+date: 2023-03-13 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#10037] Provide error highlighting in attribute macros. Note that [attribute](https://doc.rust-lang.org/reference/procedural-macros.html#attribute-macros) procedural macro expansion is disabled by default. If you want to try out, enable `org.rust.macros.proc.attr` [experimental feature](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-faq.html#experimental-features).
+
+  <img src="/assets/posts/changelog-190/macro-error-highlighting.png" width="350px"/>
+
+* [#10171] Add support for [E0224](https://doc.rust-lang.org/error_codes/E0224.html) error
```

Comment:
I guess we should add `@konrad-sz` to `MAINTAINERS` list in `changelog.py`

Reply:
#178